customer/src/main/java/no/unit/nva/customer/CustomerDoiHandler.java

```diff
@@ -93,20 +102,19 @@ private UUID toUuid(URI customerId) {
             );
     }
 
-    // TODO add this code in task #NP-27814
-    //    private ObjectMapper createJsonParser() {
-    //        var jsonFactory = new JsonFactory().configure(Feature.ALLOW_SINGLE_QUOTES, true);
-    //        var objectMapper = new ObjectMapper(jsonFactory)
-    //                               .registerModule(new ProblemModule())
-    //                               .registerModule(new JavaTimeModule())
-    //                               .registerModule(new Jdk8Module())
-    //                               .enable(DeserializationFeature.ACCEPT_EMPTY_STRING_AS_NULL_OBJECT)
-    //                               .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-    //                               .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-    //                               .setSerializationInclusion(Include.ALWAYS);
-    //        return objectMapper.disable(SerializationFeature.INDENT_OUTPUT);
-    //        //return prettyJson
-    //        //           ? objectMapper.enable(SerializationFeature.INDENT_OUTPUT)
-    //        //           : objectMapper.disable(SerializationFeature.INDENT_OUTPUT);
-    //    }
+    private ObjectMapper createJsonParser() {
```

Comment:
Can you use the object mapper from commons?

Reply:
Frontend asked for all fields, even if empty, is this a breach on policies?

Reply:
But yes, I didn't realize that I could configure mapper after creation.

Reply:
I have tried and failed, error makes no sense....